dbms/src/Interpreters/Aggregator.h

```diff
@@ -661,9 +806,17 @@ struct AggregatedDataVariants : private boost::noncopyable
     M(keys128)                                         \
     M(keys256)                                         \
     M(serialized)                                      \
-    M(multi_key_string)                                \
```

Comment:
Why delete this?

Reply:
For 2 agg keys(str,str), it has been optimized by `HashMethodFastPathTwoKeys`. It is used in tpch q1.
For other cases with GE than 3 agg keys, we may need to design and optimize later.

Reply:
OK, so for more than 3 agg string key, it now will use `serialized`?

Reply:
Yes.

Reply:
If there is any typical workload (like tpch or clickbench) use agg with 3 str keys, maybe we should add such method back.